csm/core/blogic/const.py

```diff
@@ -930,6 +930,7 @@
 RGW_ADMIN_OPERATIONS_MAPPING_SCHEMA = '{}/schema/rgw_admin_api_operations.json'.format(CSM_PATH)
 CREATE_USER_OPERATION = 'CREATE_USER'
 RGW_CLIENT_ERROR_MSG = 'Can not process the request'
+ADD_ACCESS_KEY_OPERATION = 'ADD_ACCESS_KEY'
```

Comment:
Keep the operation names similar to rgw ops. 
ADD_ACCESS_KEY_OPERATION -> CREATE_KEY